provisioning/lib/helpers

```diff
@@ -0,0 +1,26 @@
+# Usage: error_out <message> <error_code>
+error_out() {
+  echo "ERROR: ${1}"
+  exit $2
+}
+
+# Usage: get_hostname <ip address>
```

Comment:
<ip address> can be in the form of <ip address>,<ip address>,...,<ip address> - correct? If so, maybe update the usage to reflect this